universal-login-react/src/ui/Onboarding/Onboarding.tsx

```diff
@@ -46,34 +33,55 @@ export const Onboarding = (props: OnboardingProps) => {
 
   useEffect(() => {
     setImmediate(async () => {
-      const wallet = await walletCreationService.deployWhenReady(() => modalService.hideModal());
-      modalService.hideModal();
+      const wallet = await walletCreationService.deployWhenReady();
       props.onCreate?.(wallet);
     });
   }, []);
 
   return (
     <div className="universal-login">
       <div className={getStyleForTopLevelComponent(props.className)}>
-        <ReactModalContext.Provider value={modalService}>
-          <div className="perspective">
-            <WalletSelector
-              sdk={props.sdk}
-              onCreateClick={onCreateClick}
-              onConnectClick={onConnectClick}
-              domains={props.domains}
-              tryEnablingMetamask={props.tryEnablingMetamask}
-              actions={[WalletSuggestionAction.connect, WalletSuggestionAction.create]}
+
+        <MemoryRouter initialEntries={['/selector']}>
```

Comment:
Maybe we should make it constant and move it to `constants/ui`. Let's merge this PR and decide about extracting it in the next.